documentation/content/Supported Plugins/multi-part-plugin.md

```diff
@@ -1,69 +1,70 @@
-Title: How to use Multi Part plugin
+Title: Add a Series Indicator to Your Multipart Articles
 Tags: pelican-theme, pelican-plugin, navigation, web-design
 Category: Supported Plugins
 Date: 2014-04-20 18:18
 Slug: how-to-use-multi-part-plugin
 Comment_id: 3ws2xke-how-to-use-multi-part-plugin
 Subtitle:
-Summary: Elegant integrates with Multi Part plugin out of the box
+Summary: Elegant can be configured to provide a series section on the right sidebar. Only visible in articles that are labelled as part of a series, this indicator allows navigation between the articles in the series.
 Keywords:
+Authors: Talha Mansoor, Jack De Winter
 
-[Multi
-parts](https://github.com/getpelican/pelican-plugins/tree/master/multi_part) is
-a useful plugin that lets you write "multi-part" articles.
+When writing articles about certain topics, it is advantageous to split a single article into
+multiple articles. Without splitting the article up, the author would be forced to cram all
+of the content into a denser and much longer article, reducing its effectiveness and
+readability in the process. Splitting the article allows the author to focus on a specific
+concept of the larger article, thereby increasing the overall appearance and readability.
 
-To mark articles that belong to the same series, define `parts` metadata.
+Elegant provides the ability to present a view of the articles in the series in the middle of
+the right sidebar. This section starts with the name of the series, followed by one bullet
+point for each of the articles in the series. The text for the article is prefaced with
+"Part N: " (where N is the index of the article) and then the title for the article. To make
+navigation easier, the current article is presented in italics, with the other articles being
+presented as links to their respective articles.
 
-    :::rest
-    :parts: iCloud 101 Series
+Here is an example of what the Series section may look like:
 
-Elegant shows the multi-part series in the sidebar.
+![series example in the sidebar]({filename}/images/elegant-theme_multi-part-sidebar.png)
```

Comment:
Instead of filename it should be `static`.

Reply:
Done.  Also scanned any other file I have changed, and changed it there as well.

There are a number of other references in other areas.  I'll change those as I get to those other documents.